scripts/extractions/extract_meteo.py

```diff
@@ -0,0 +1,162 @@
+"""Extract AGERA5 (Meteo) data using OpenEO-GFMAP package."""
+import argparse
+from functools import partial
+from pathlib import Path
+
+import geojson
+import geopandas as gpd
+import openeo
+import pandas as pd
+from extract_sar import (
+    _buffer_geometry,
+    _filter_extract_true,
+    _pipeline_log,
+    _setup_logger,
+    _upload_geoparquet_artifactory,
+    create_job_dataframe,
+    generate_output_path,
+)
+from openeo_gfmap import Backend, TemporalContext
+from openeo_gfmap.backend import vito_connection
+from openeo_gfmap.manager.job_manager import GFMAPJobManager
+from openeo_gfmap.manager.job_splitters import (
+    _append_h3_index,
+    _load_s2_grid,
+    split_job_s2grid,
+)
+
+
+def create_datacube_meteo(
+    row: pd.Series,
+    connection: openeo.DataCube,
+    provider=None,
+    connection_provider=None,
+    executor_memory: str = "2G",
+    executor_memory_overhead: str = "1G",
+) -> gpd.GeoDataFrame:
+    start_date = row.start_date
+    end_date = row.end_date
+    temporal_context = TemporalContext(start_date, end_date)
+
+    # Get the feature collection containing the geometry to the job
+    geometry = geojson.loads(row.geometry)
+    assert isinstance(geometry, geojson.FeatureCollection)
+
+    # Filter the geometry to the rows with the extract only flag
+    geometry = _filter_extract_true(geometry)
+    assert len(geometry.features) > 0, "No geometries with the extract flag found"
+
+    # Performs a buffer of 64 px around the geometry
+    geometry_df = _buffer_geometry(geometry, distance_m=5)
+    spatial_extent_url = _upload_geoparquet_artifactory(geometry_df, row.name)
+
+    bands_to_download = ["temperature-mean"]
+
+    cube = connection.load_collection(
```

Comment:
are we doing a rescaling of meteo variables somewhere? We should convert to integers to save storage (I can point you at default scaling)

Reply:
@kvantricht can you point me at that scaling?

Reply:
This is done